core/src/empty/empty.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import { storiesOf } from "@storybook/react";
+import { Empty } from "../empty/empty";
+
+storiesOf("Empty", module).add("Main", () => (
+	<div>
+		<Empty active={false} errorMsg="Lỗi kết nối, vui lòng chờ trong giây lát." />
+		<br />
+		<Empty active={true} errorMsg="Lỗi kết nối, vui lòng thử lại sau" onClickBtn={alertRetry}/>
+	</div>
+));
+
+let alertRetry = () => {
+	alert("retry connection !");
```

Comment:
Moai have alternative [alert](https://docs.moaijs.com/?path=/docs/components-dialog--utilities#utilities). I think you can use Moai's alert here.